fcl/dunefd/g4/supernova_g4_halfActiveVol_dune10kt_1x2x6.fcl

```diff
@@ -0,0 +1,13 @@
+# This fhicl enables the generation of energy deposits in just half the active volume (x>0) to emulate the lateral APA in the HD geometry
```

Comment:
Could this fcl inherit from the supernova fcl?  Then only a small number of overrides need to be implemented here

Reply:
Absolutely :-)